pkg/matcher/README.md

```diff
@@ -21,19 +27,30 @@ Short Syntax
 
  Long Syntax
 
-     <line>      ::= [ <not> ] <format> <separator> <expr>
+     [ <not> ] <format> <separator> <expr>
      <format>    ::= [ 'string' | 'glob' | 'regexp' | 'simple_patterns' ]
```

Comment:
remove a new line after `Long Syntax` and add after `[ <not> ] <format> <separator> <expr>`